ACD-618 (#30)
Actindo-AG/Documentation: Channels/UserInterface/03b_Errors.md

[!! Check the failed offer uploads](../Operation/03_CheckOfferUpload.md#check-the-failed-offer-uploads)
[!! Retry a faulty offer upload](../Troubleshooting/01_RetryFaultyUpload.md)

# Errors

*Omni-Channel > Offer upload > Tab ERRORS*

![Errors](../../Assets/Screenshots/Channels/OfferUpload/Errors/ExportErrors.png "[Errors]")

**Export errors**

- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   

>   Click this button to display the search bar and search for a scheduled offer upload.

Reviewer: ```suggestion
  Click this button to display the search bar and search for an export error.
```

- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   

>   Click this button to update the list of scheduled offer uploads.

Reviewer: ```suggestion
  Click this button to update the list of export errors.
```

[comment]: <> (Stimmt das?Wo wähle ich das aus/ordne ich das zu? Workflows?)

- *Log Message Title*    
  Title of the error log message.

- *Log Message*   
  Complete log message to the error. Click the corresponding row of the error to display the *Detail of log message "Log message title"* view and check further details of the error.

- *Finally Failed*   
  Indication whether the offer upload finally failed or not:
  - ![Check](../../Assets/Icons/Check.png "[Check]") (Checkmark): The offer upload finally failed.
  - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross): The offer upload can be retried.

- *ID*  
  Offer identification number. The ID number is automatically assigned by the system after creation.

- *Created on*  
  Date and time when the error occurred.

- *Created by*  
  Name and username of the user who changed the offer and triggered the upload.

- *Modified on*  
  Date and time of the last modification.

- *Modified by*  
  Name and username of the user who modified the offer.

[comment]: <> (Spaltenbeschreibung prüfen - was bezieht sich auf den Error und was auf das offer selbst?)

## Detail of log message "Log message title"

> *Omni-Channel > Offer import > Tab ERRORS > Select export error*

Reviewer: ```suggestion
*Omni-Channel > Offer upload > Tab ERRORS > Select export error*
```
